kitchen.resources-install.yml

```diff
@@ -296,3 +296,27 @@ suites:
         - recipe:aws-parallelcluster-test::docker_mock
         - resource:package_repos
       resource: lustre
+  - name: network_service_restart
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-common::test_resource]
+    verifier:
+      controls:
+        - network_service_running
+    attributes:
+      dependencies:
+        - resource:node_attributes
```

Comment:
Nitpick: this is redundant

Reply:
Ah right, I had this before I switched to the tests::setup.  I was following other tests I'd seen.  Overall this process has been quite confusing deciding which resources and run lists to use, since there seems to be alot of duplication across the tests